src/pages/components/CertificateActionsButtons.js

```diff
@@ -66,41 +43,45 @@ export default function CertificateActionsButtons({
 const Sidebar = styled(Grid.Panel)`
   align-items: center;
   justify-items: center;
-  min-width: 340px;
   color: white;
   background: #0c3b38;
 `
 
 const PaddedDiv = styled.div`
-  padding: 34px 21px;
   width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 34px 21px;
 `
 
 const SmallButton = styled(Button)`
-  width: 100%;
-  min-width: 132px;
-  height: 100% !important;
```

Comment:
I think I can say I do hate it/hated that `!important` when I was basically unable to use Button

Reply:
important is evil and had I spotted it in review it would not have been allowed